cluster/ubuntu/util.sh

```diff
@@ -231,6 +231,11 @@ EOF
 }
 
 function create-kubelet-opts() {
```

Comment:
My ambition here was only to take the first baby step, which is to add CNI support to the Ubuntu scripting.  My plan was to think bigger after that is accomplished.  I myself have not been using local-up-cluster and am not familiar with it.  Do we really need to take the bigger step right out of the gate?